File: src/main/java/org/usfirst/frc/team6325/robot/commands/Drive/ResetGyro.java
package org.usfirst.frc.team6325.robot.commands.Drive;

import edu.wpi.first.wpilibj.command.Command;
import org.usfirst.frc.team6325.robot.Robot;


public class ResetGyro extends Command 
{
	public ResetGyro() 
	{
		// Use requires() here to declare subsystem dependencies
		requires(Robot.drivetrain);
	}

	// Called just before this Command runs the first time
	@Override
	protected void initialize() 
	{
		Robot.drivetrain.resetGyro();
	}

	// Make this return true when this Command no longer needs to run execute()
	@Override
	protected boolean isFinished()
	{
		return true;
	}
}

File: src/main/java/org/usfirst/frc/team6325/robot/commands/Drive/ResetGyro.kt
package org.usfirst.frc.team6325.robot.commands.Drive

import org.sertain.command.*
import org.usfirst.frc.team6325.robot.Polybius


public class ResetGyro : Command(), SendableBase
{
    init { requires Polybius.drivetrain }

    override fun execute() = Polybius.drivetrain.resetGyro()
    override fun onDestroy() = return true
}